README.md

```diff
@@ -1,62 +1,62 @@
-# school-library
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-> School Library.
-
-## Built With
-
-- Ruby
-
-## Getting Started
-
-To get a local copy up and running follow these simple example steps.
-
-- copy this link git@github.com:NatiG25/school-library.git
-- get the directory that you want to clone the repository.
-- open the command prompt in this directory.
-- write git clone git@github.com:NatiG25/school-library.git
-- go to the repository folder in your command prompt cd school-library
-- use npm install to install the dependencies.
-
-## Prerequisites
-
-- Have a working and updated browser
-- Have a local version control like git install on your computer
-- Have an account and sign in on GitHub, as online/remote version control system
-
-### Setup
-
-- Clone the git repo using this command: git clone git@github.com:NatiG25/school-library.git
-
-### Run tests
-
-- rubocop --color : To test ruby codes
-
-## Live Demo
-
-- [Live Demo Link]()
-
-## Author
-
-- GitHub: [@NatiG25](https://github.com/NatiG25)
-- Twitter: [@NatiG87702270](https://twitter.com/NatiG87702270)
-- LinkedIn: [@Natnail](https://linkedin.com/in/temesgen-g-gorgis-0910a6229 )
-
-## Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the issues page.
-
-## Show your support
-
-Give a star if you like this project!
-
-## Acknowledgments
-
-- Microverse linters
-- Coding partners
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
+# school-library
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> School Library.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link git@github.com:NatiG25/school-library.git
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write git clone git@github.com:NatiG25/school-library.git
+- go to the repository folder in your command prompt cd school-library
+- use npm install to install the dependencies.
+
+## Prerequisites
+
+- Have a working and updated browser
+- Have a local version control like git install on your computer
+- Have an account and sign in on GitHub, as online/remote version control system
+
+### Setup
+
+- Clone the git repo using this command: git clone git@github.com:NatiG25/school-library.git
+
+### Run tests
+
+- rubocop --color : To test ruby codes
+
+## Live Demo
+
+- [Live Demo Link]()

```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove any redundant link from your ReadMe. This will make your ReadMe more professional